### ISSUE TITLE
Which client software to use for windows?

### ISSUE THREAD

**Issue by skuskusku** [dorssel/usbipd-win#906]
Hello Frans,

it is unclear to me if usbipd-win is suitable for my use case: I want to remote a digicert code signing usb token to a windows build vm. I don't care if the server side is running on windows or linux, but I need client side access from a windows 10 build vm to the server running usbipd-win or similar. 

What I haven't understood is: Does the downloadable MSI of yours only supply the server side of USB-remoting or does it contain the necessary software for clients as well? If not, can you recommend a client package for windows? From my investigations so far, all IP/USB windows based drivers are in not so a mature state, lots haven't been updated for ages and some lack properly signed drivers. Can you share your experience? Stability is of uttermost importance for my build VMs.

Any help appreciated,

-- 
Stefan

**Comment by dorssel** [comment]
You are correct. `usbipd-win` is only the server side of the protocol. Client tooling for Windows appears to exist; some users reported success. But I don't use Windows clients myself, so I wouldn't know. As reported in other issues, and in the README, client side is outside the scope of this project. Maybe search through the older (possibly closed) issues/discussions for user stories.
